### DISCUSSION TITLE
what happened to the m3u8 link for nhk educational tv

### DISCUSSION THREAD

**Discussion by dantheman121** [iptv-org/community#465] [Q&A] [answered]
i remember there being a m3u8 link in the database and i remember watching gigantosaurus on it so why did it get removed?

**Comment by dantheman121** [comment]
@freearhey

**Comment by PopeyeTheSai10r** [comment, marked as answer]
Hey, It looks like the link stopped working and was removed. https://github.com/iptv-org/iptv/issues/6064
